src/app/constants/browserStorage.ts

```diff
@@ -3,7 +3,8 @@
  * Licensed under the MIT License
  **********************************************************/
 export const PRIVATE_REPO_CONNECTION_STRING_NAME = 'PRIVATE_REPO_CONNECTION_STRING_NAME';
-export const CONNECTION_STRING_NAME = 'CONNECTION_STRING_NAME';
+export const CONNECTION_STRING_LIST_MAX_LENGTH = 5;
+export const CONNECTION_STRING_NAME_LIST = 'CONNECTION_STRING_NAME'; // store hub connection strings in localStorage separated by comma
```

Comment:
nit: Should this be CONNECTION_STRING_LIST_NAME?

Reply:
I wanted to change it as well, but I think we are "married" to old value now, as we can't break the experiecen for people who used the old version before

Reply:
I think we'd have _both_ `CONNECTION_STRING_NAME` to hold the selected connection string from the `CONNECTION_STRING_LIST` values.

Reply:
i don't think it is necessary though. because
1, it is duplicate information. the first in the list if what user currently selected, we only need the selector to extract information out
2. We would have to maintain both of the value in sync
So I am not going to add both of the value just for the sake of the naming convention, If necessary I can revert the constant's name back.

Reply:
or if we are ok with breaking the users just once, we can rename it

Reply:
If we do, we should tag with Breaking Change.

Reply:
yeah that's true. probably not worth doing it?

Reply:
I'd still do it and just tag it. Better to do it early. And it's really a mild inconvenience of having to put the connection string back in. Not like it is taking away functionality.